python/ray/experimental/sgd/pytorch/examples/example-sgd.yaml

```diff
@@ -37,18 +36,18 @@ head_node:
 worker_nodes:
     InstanceType: p3.8xlarge
     ImageId: ami-0757fc5a639fe7666
+    # Run workers on spot by default. Comment this out to use on-demand.
     InstanceMarketOptions:
         MarketType: spot
```

Comment:
perhaps a comment somewhere that using spot instances and `max_retries=0` could end poorly

Reply:
Hm, I think this belongs not in the cluster specification but rather in the python code? 

Won't do this for now but will keep in mind.